src/connector/src/parser/unified/util.rs

```diff
@@ -24,8 +24,8 @@ pub fn apply_row_operation_on_stream_chunk_writer_with_op(
 ) -> AccessResult<()> {
     let f = |column: &SourceColumnDesc| row_op.access_field(column);
     match op {
-        ChangeEventOperation::Upsert => writer.insert(f),
-        ChangeEventOperation::Delete => writer.delete(f),
+        ChangeEventOperation::Upsert => writer.do_insert(f),
+        ChangeEventOperation::Delete => writer.do_delete(f),
```

Comment:
Looks like `apply_row_operation_on_stream_chunk_writer_with_op`'s only caller now is `apply_row_operation_on_stream_chunk_writer` below.